tests/TestCodeGen.hs

```diff
@@ -254,6 +254,7 @@ compileTestDotProtos logger recStyle decodedStringType = do
   let protoFiles :: [Turtle.FilePath]
       protoFiles =
         [ "test_proto.proto"
+        , "test_proto_empty_field.proto"
```

Comment:
This will trigger code generation, and perhaps that's all you care about: does it parse without error?  But if you wanted to actually test the generated code's interoperability with generated Python code then you would have to also modify:
  * `tests/SimpleDecodeDotProto.hs`
  * `tests/SimpleEncodeDotProto.hs`
  * `tests/check_simple_dot_proto.py`
  * `tests/send_simple_dot_proto.py`

But for something like empty statements I would think you could do a much simpler check: parse the .proto file into AST and then just check that parsing is successful and that the AST contains message types with the expected fields, both before and after the empty statement.

Reply:
Thanks for the revised tests.